### ISSUE TITLE
Adding a message at the end of all message list for user awareness.

### ISSUE THREAD

**Issue by harshitagupta30** [zulip/zulip-android#344]
Just a suggestion can we add a message like "No new messages" at the end of the message list when the user is trying to update the list to make user aware that list has been updated instead of the present scenario shown in gif.

![videotogif_2017 01 29_20 24 52](https://cloud.githubusercontent.com/assets/12701036/22405034/2a54e128-e662-11e6-93b1-a5723d4a75c7.gif)

@timabbott @kunall17 @Sam1301 Hi, please review!

**Comment by yadav-rahul** [comment]
@harshitagupta30 I don't think that adding "No new messages" at the end of list(while updating) would be a good option because How can one predict that whether new messages are there or not while updating the list?

**Comment by harshitagupta30** [comment]
Since we're fetching the messages from API and if all messages have been loaded and after that user tries to fetch by scrolling then a message should pop out that all messages has been fetched.

**Comment by kunall17** [comment]
I thought of implementing a snack bar or toast 
while writing the recycler view but the problem is it can be spammy as well! And may not look good. 

What are the other options for notifying?

**Comment by harshitagupta30** [comment]
Hi @kunall17! 
In my opinion, it doesn't look spammy. Assuming one has low internet connection and he is trying to load new messages which is taking time and if they are not loaded he'll think the internet is not working properly while the case is all the messages has been loaded. So if we can provide a message it will let user know that all messages have been fetched.  
In fact I tried to implement that and it looks good to me. I am attaching the gif as well. 

![videotogif_2017 01 30_18 46 42](https://cloud.githubusercontent.com/assets/12701036/22424610/410c0944-e71d-11e6-950f-fee78273c590.gif)

What's your opinion?

I would love to hear your thoughts on this! @timabbott @Sam1301

**Comment by kunall17** [comment]
@harshitagupta30 Yeah i can connect very well, and understand the problem! 
I also asked what can be other options rather than snackbar, if no other good suggestions we'll have to go with snackbar!  

Can you upload the code (maybe a .patch file) or a commit, I wanted to test?

**Comment by harshitagupta30** [comment]
I tested with toast and alert dialog box as well but this the best possible way and very clean, in my opinion. 
Yeah! I will upload the .patch file for the testing.

**Comment by Sam1301** [comment]
@harshitagupta30 feel free to claim this task using zulipbot. That way it will be helpful for others to skip through issues which are not already worked on :)

**Comment by Sam1301** [comment]
@harshitagupta30 assigning you for now since you already created a pr for it :+1: :)

**Comment by abdulwd** [comment]
I would like to work on this issue.
